src/main/java/com/redhat/cloud/notifications/db/NotificationResources.java

```diff
@@ -41,9 +40,9 @@
                             .bind("$5", history.getEventId());
 
                     if (history.getDetails() != null) {
-                        st.bind("$6", Json.of(new JsonObject(history.getDetails()).encode()));
+                        st.bind("$6", new JsonObject(history.getDetails()).encode());
```

Comment:
Just wondering, no change is required to "load" from the database?

Reply:
The `details` field was already loaded as a String [here](https://github.com/RedHatInsights/notifications-backend/blob/8a5f888055194825c492ec0b2a93d5c623760110/src/main/java/com/redhat/cloud/notifications/db/NotificationResources.java#L101) so there's nothing to change.